README.md

```diff
@@ -72,10 +72,6 @@ The rails-blog-app is a classic example of a blog website. It is a fully functio
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 
-## 🚀 Video Project Presentation <a name="video-project-presentation"></a>
-
-- NA
-
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
![image](https://github.com/codeWithHyder/blog-rails/assets/93039359/8fa21e6f-b9e6-4a7a-9cdc-1ee158573be3)

- [ ] It would be nice if you fix the highlighted links in the image above 🤔 because they are broken right now 😨 Kindly fix them or remove them if you don't have any sections for them to make your readme descriptive and professional 👍